api/response.go

```diff
@@ -41,12 +41,14 @@ func (r *Response) Error() error {
 
 	r.Body.Close()
 	r.Body = ioutil.NopCloser(bodyBuf)
+	ns := r.Header.Get("X-Vault-Namespace")
```

Comment:
Can you use consts.NamespaceHeaderName instead of the literal string?

Reply:
Done